.github/workflows/release.yml

```diff
@@ -7,6 +7,9 @@ on:
 
 jobs:
   goreleaser:
+    permissions:
+      id-token: write
+      contents: write
```

Comment:
Just to confirm: This needs `contents: write` to be able to add things to the release right?